third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1387,14 +1414,17 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 			}
 		}
 
-		if d.HasChange(prefix + ".access_config") {
-
-			// TODO: This code deletes then recreates accessConfigs.  This is bad because it may
-			// leave the machine inaccessible from either ip if the creation part fails (network
-			// timeout etc).  However right now there is a GCE limit of 1 accessConfig so it is
-			// the only way to do it.  In future this should be revised to only change what is
-			// necessary, and also add before removing.
+		readFingerPrint := func() error {
```

Comment:
it feels like a function called readFingerPrint should return the fingerprint, no? This is really reading the instance.

Also fingerprint is one word so it would be readFingerprint :)